osu.Game.Tournament/Components/DateTextBox.cs

```diff
@@ -12,24 +10,21 @@ namespace osu.Game.Tournament.Components
 {
     public partial class DateTextBox : SettingsTextBox
     {
-        public new Bindable<DateTimeOffset> Current
+        private readonly BindableWithCurrent<DateTimeOffset> current = new BindableWithCurrent<DateTimeOffset>();
```

Comment:
This doesn't fix any crashes, just noticed it was a pretty dodgy / dated implementation method.